packages/kuma-gui/vite.config.production.ts

```diff
@@ -37,7 +38,8 @@ export const config: UserConfigFn = () => {
         },
       }),
       fakeApi({
-        fs,
+        mockEndpointArgs: dependencies,
+        mockEndpointsDictionary: fs,
```

Comment:
we are gonna cvall these user facing props, dependencies and fs (based on the original approach the fakeApi plugin)